backend/fms_core/migrations/0038_v3_9_0.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 3.1 on 2021-10-21 21:20
+
+from django.conf import settings
+from django.db import migrations, models
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+
+def create_library_qc_objects(apps, schema_editor):
+    Protocol = apps.get_model("fms_core", "Protocol")
+    PropertyType = apps.get_model("fms_core", "PropertyType")
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        admin_user_id = admin_user.id
+
+        reversion.set_comment("Create objects related to protocol Library Quality Control")
+        reversion.set_user(admin_user)
+
+        # Create PropertyType and Protocols
+        PROPERTY_TYPES_BY_PROTOCOL = {
+            # note: property type values must be unique so these all need the Library prefix
```

Comment:
This is the case but Sebastian is changing that in his PR for library conversion. You can keep the same names. When Sebastian merge his branch, the unique constraint will be with both the name and the protocol id so they'll be unique.